src/api/posts/src/index.js

```diff
@@ -1,9 +1,17 @@
-const { Satellite } = require('@senecacdot/satellite');
+const { Satellite, Redis } = require('@senecacdot/satellite');
 
 const posts = require('./routes/posts');
 
-const service = new Satellite();
+const redis = Redis();
+
+const service = new Satellite({
+  healthCheck: () => {
```

Comment:
You're not returning the promise here.  Should be: `healthCheck: () => redis.ping()`